deploy/charts/templates/rbac.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "openebs-ndm.serviceAccountName" . }}
+{{- end }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
check if this can be moved to v1. Looks like v1beta1 will be deprecated soon.

Reply:
done